description.md

```diff
@@ -38,6 +38,73 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Assignments
 
+Assignments are unassisted practical activities that learners undertake individually or in teams over a longer timespan.
+The primary objective of an assignment is to create a compelling scenario that allows learners to apply the concepts they have learned.
+
+#### Storing Assignments
```

Comment:
This goes to the `Projects` subsection in the `Types of Content` section. Here you talk about teaching stages: launching, providing support, grading assignments, i.e. what happens during an actual semester - unlike I know we typically do, the assumption is that all materials are ready in the repository **before** the start of the semester.